tracing-subscriber/src/registry/sharded.rs

```diff
@@ -97,9 +115,29 @@ impl Registry {
     fn get(&self, id: &Id) -> Option<Guard<'_, DataInner>> {
         self.spans.get(id_to_idx(id))
     }
+
+    pub(crate) fn ref_guard(&self, id: Id) -> LayerGuard<'_> {
```

Comment:
I don't find this name super descriptive of what this method _means_ — while I understand what's going on here, I don't think this would be very clear to a new contributor reading this code for the first time. 

What if we changed this name to something that explained _why_ we call this, rather than what it returns? Maybe `start_close` or something? Of course, if you can think of a better name, that would be great.

A comment explaining what this does could be good as well.

Reply:
Added details and renamed to `start_close`. I couldn't think of anything better :)